src/client/components/Input/Input.css

```diff
@@ -0,0 +1,12 @@
+.input-components {
```

Comment:
Should be just `.input` or `.input-component`. Not `components` - it means _multiple_ inputs.